api/utils/http_test.go

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) 2018 The VeChainThor developers
+
+// Distributed under the GNU Lesser General Public License v3.0 software license, see the accompanying
+// file LICENSE or <https://www.gnu.org/licenses/lgpl-3.0.html>
+
+package utils
```

Comment:
Does utils_test work?

Reply:
Yes, it does. In this case I'm only using public functions from utils package. I was reading this [page](https://go.dev/doc/tutorial/add-a-test) and here it suggests to put tests in the same package as the implementation. I've realized there are different ways to approach this, do we have any convention we're following?

Reply:
I was under the impression that the package with the suffix `_test` would not be compiled into the official binary so when I was doing tests, if I don't use the private functions I would use `xxx_test` as the package name.